Firestore/core/src/firebase/firestore/core/filter.h

```diff
@@ -98,6 +100,9 @@ inline bool operator!=(const Filter& lhs, const Filter& rhs) {
   return !(lhs == rhs);
 }
 
+/** A list of Filters, as used in Queries and elsewhere. */
+using FilterList = immutable::AppendOnlyList<std::shared_ptr<Filter>>;
```

Comment:
Can this be `std::shared_ptr<const Filter>`?

Reply:
Yes! I thought this change would have a far larger blast radius than it turned out it had. Done.